PLOS/allofplos
# Update to use cloudflare storage of corpus zip

Changes:
- Download files in parallel (to help speed things up, esp. on the k8s cronjob that updates and uploads the zip)
- Support downloading the zip file from the new cloudflare system, rather than gdrive
- Remove some old code

To test:

```
$ pipenv install
...
$ pipenv run python -m allofplos.update
...
```

It should download the zip file
```
Not enough articles in /home/egh/c/allofplos/allofplos/allofplos_xml, re-downloading zip file
Creating folder for article xml
...
```

Check that it's connecting to cloudflare:
```
🤔 ps -fade |grep allofplos
egh       871858  842183 15 21:50 pts/1    00:01:06 /home/egh/.local/share/virtualenvs/allofplos-F990z4Rq/bin/python -m allofplos.update
🤔 sudo lsof |grep 871858 |grep TCP
python    871858                               egh    4u     IPv4            4334030        0t0        TCP ehetzner-thinkpad-t470.lan:48522->172.67.72.54:https (ESTABLISHED)
python    871858 871872 python                 egh    4u     IPv4            4334030        0t0        TCP ehetzner-thinkpad-t470.lan:48522->172.67.72.54:https (ESTABLISHED)
🤔 dig +noall +authority -x 172.67.72.54
67.172.in-addr.arpa.	1215	IN	SOA	cruz.ns.cloudflare.com. dns.cloudflare.com. 2288625501 10000 2400 604800 3600
```
